### ISSUE TITLE
Train using pretrained weights from pretrain command

### ISSUE THREAD

**Issue by CatarinaPC** [explosion/spaCy#5225]
I'm training my own French NER models using SpaCy and evaluating them along with SpaCy's French NER pre-trained models.

I wanted to experiment with the SpaCy `pretrain` command and see if I can improve my models. I have now ran this command on raw texts. It ran for 1000 iterations, so "technically", I have 1000 models.

Which of these models should I give as the parameter for `--init-tok2vec` when I run the `train` command ?

**Comment by svlandeg** [comment]
Usually, the loss keeps more or less decreasing, so you could just take the model from the last iteration. Alternatively, you can inspect the `log.jsonl` file that is included in your output directory, and take the model from the iteration which has the lowest `epoch_loss`.
